pylint/lint.py

```diff
@@ -595,6 +595,10 @@ def error_mode(self):
             for msg_id in self._checker_messages('python3'):
                 if msg_id.startswith('E'):
                     self.enable(msg_id)
+            config_parser = self.cfgfile_parser
+            if config_parser.has_option('MESSAGES CONTROL', 'disable'):
```

Comment:
I don't understand why this is here and what's the intention with it? Can you elaborate what's the reasoning behind this change?

Reply:
`self.disable('all')` followed by selectively enabling all `python3` error messages negates the effect of any `[MESSAGES CONTROL] disable=...` line in the config file (which may be disabling some python3 errors. This re-applies them (if they exist).